Lib/importlib/_bootstrap_external.py

```diff
@@ -1552,6 +1576,10 @@ def _setup(_bootstrap_module):
         SOURCE_SUFFIXES.append('.pyw')
         if '_d.pyd' in EXTENSION_SUFFIXES:
             WindowsRegistryFinder.DEBUG_BUILD = True
+    bytecode_path = _os.environ.get(b'PYTHONBYTECODEPATH') or None
```

Comment:
I still think it be worth looking into doing this as part of the runtime init (i.e. PEP 432).  It's the approach we follow for similar environment variables (and CLI flags).  Addressing this would impact files like Include/internal/pystate.h, Modules/main.c, and Python/pystate.c.

Reply:
I can look at that, though I'm not clear what concrete benefit it would bring, unless we want to add a corresponding CLI arg.